torch_xla/csrc/convolution.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef XLA_TORCH_XLA_CSRC_CONVOLUTION_H_
```

Comment:
Why not using `#pragma once` anymore?

Reply:
We had header guards in one part of the code and pragma in the other.

Since google code style recommends header guards (https://google.github.io/styleguide/cppguide.html#The__define_Guard), and discourages the `#pragma once` even on windows (https://google.github.io/styleguide/cppguide.html#Windows_Code), I went for the header guards. They are standardized and recommended.

I have no opinion really outisde of these guidances.